README.md

```diff
@@ -5,3 +5,17 @@ from stdin and writes it to stdout. Can also be run as a web service by passing
 the `-addr` option.
 
 ![PARTY](/out.gif)
+
+# Install
+
+```
+$ go get github.com/disintegration/imaging
+$ go get github.com/akenn/graphics-go/graphics
+```
+
+# Usage
+
+```
+$ go run index.go main.go --addr=localhost:8000
```

Comment:
should just be `-addr`, also you should note that this is just for the mode where party-aas runs as a webserver, it also works as a normal command-line utility that reads from stdin.